vaadin-upload-flow-parent/vaadin-upload-flow/src/main/java/com/vaadin/flow/component/upload/receivers/FileBuffer.java

```diff
@@ -38,11 +38,17 @@
 public class FileBuffer extends AbstractFileBuffer implements Receiver {
 
     private FileData file;
+	
+	public FileBuffer(){}
```

Comment:
Missing a `super();` call here, otherwise the default file factory will not be initialized.

Also missing API docs, which could be copied from `AbstractFileBuffer`.